lib/systems/default.nix

```diff
@@ -50,7 +50,7 @@ rec {
         else if final.isFreeBSD             then "fblibc"
         else if final.isNetBSD              then "nblibc"
         else if final.isAvr                 then "avrlibc"
-        else if final.isNone                then "newlib"
+        else if final.isUefi                then "none" # TODO: execute can actually be QEMU?
```

Comment:
I don't understand what this `TODO` is saying.

Reply:
Can we emulate UEFI programs on QEMU? I guess we could have a remote builder for UEFI if we had baremetal compilers in UEFI, etc. :D ?

Reply:
> Can we emulate UEFI programs on QEMU?

Probably.  I just can't figure out what this has to do with the choice of `libc`.